lib/rails_erd/domain.rb

```diff
@@ -120,7 +120,32 @@ def specializations_mapping
     end
 
     def models
-      @models ||= @source_models.select { |model| check_model_validity(model) }.reject { |model| check_habtm_model(model) }
+      @models ||= @source_models
+                    .reject { |model| tableless_rails_models.include?(model) }
+                    .select { |model| check_model_validity(model) }
+                    .reject { |model| check_habtm_model(model) }
+    end
+
+    # Returns Rails model classes defined in the app
+    def rails_models
+      %w(
+        ActionMailbox::InboundEmail
+        ActiveStorage::Attachment
+        ActiveStorage::Blob
+        ActiveStorage::VariantRecord
+        ActionText::RichText
+        ActionText::EncryptedRichText
+      ).map{ |model| Object.const_get(model) rescue nil }.compact
```

Comment:
This isn't a blocker per se, but I wonder if there's a more automated way to retrieve Rails classes that are clogging us up here.. for future iterations, I suspect.

Reply:
It's a good point as this list would need to be updated for newer versions of Rails. I couldn't think of any solutions on how to do this efficiently though.

This method works for older versions of Rails that don't have these models.